frontend/src/App.js

```diff
@@ -2,9 +2,45 @@ import React, { Component } from 'react';
 import './App.css';
 
 class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      quote: 'They don\'t know that we know they know we know.',
```

Comment:
this is a nit, but it will come up a lot when you're doing frontend engineering. A lot of designers want you to use apostrophe `’` rather than single quote `'` in human facing text